bundle/Generator/ConfigurationGenerator.php

```diff
@@ -54,17 +64,24 @@ public function generate(InputInterface $input, OutputInterface $output): void
             $settings['ezpublish']['system']['frontend_group']['translation_siteaccesses'] = $siteAccessNames;
         }
 
+        $settings['ezpublish']['system']['frontend_group']['content']['tree_root']['location_id'] = '%ngsite.default.locations.tree_root.id%';
+
+        $designType = $input->getOption('site-design') === self::LOCAL_DESIGN ? 'app' : 'remote';
+
         $settings['netgen_layouts']['design_list']['app'] = ['app'];
-        $settings['netgen_layouts']['system']['frontend_group']['design'] = 'app';
+        $settings['netgen_layouts']['design_list']['remote'] = ['remote', 'app'];
```

Comment:
Shouldn't we add this one only if `$designType` is set to `remote`? Same hoes for `ezdesign` below.

Reply:
Well, I do not think to. My thoughts are that we should keep this is an option, as we will have the templates for that design already ready. So if anyone will ever want to switch to `remote` design further down the line, they would only need to change it in one place, while the design itself would already be prepared.

Reply:
Hm, okay, makes sense!